docs/examples/testing/integration_test.py

```diff
@@ -19,25 +21,26 @@ def remote_method(self, value):
 
 
 class ServiceY(object):
+    """ Service under test
+    """
     name = "service_y"
 
     @rpc
     def append_identifier(self, value):
         return "{}-y".format(value)
 
-# =============================================================================
-# Begin test
-# =============================================================================
 
-def test_service_integration():
+def test_service_x_y_integration():
+
+    # run services in the normal manner
     config = {'AMQP_URI': 'amqp://guest:guest@localhost:5672/'}
     runner = ServiceRunner(config)
     runner.add_service(ServiceX)
     runner.add_service(ServiceY)
     runner.start()
```

Comment:
probably worth making clear that we are using hooks to fire x, but the interaction between x and y (which we are testing) is just as if under normal operation